package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Abstraction",
+  "name": "abstraction",
```

Comment:
Repo name is camel case, tho? Shouldn't it match?

Reply:
I had thought the name property for npm packages was restricted to lowercase. I'll doubblecheck